CustomServer/swagger/swagger.yml

```diff
@@ -45,7 +45,7 @@ paths:
           required: false
```

Comment:
what does required false means here ?

Reply:
its mean these fields are optional. we can hit api even without providing these query parameters